ui/src/plot.tsx

```diff
@@ -1060,9 +1064,41 @@ export const
           space = spaceTypeOf(raw_data, marks),
           data = refactorData(raw_data, plot.marks),
           { Chart } = await import('@antv/g2'),
+          tooltipCfg = Object.keys(data[0]).join('*'),
           chart = plot.marks ? new Chart(makeChart(el, space, plot.marks, model.interactions || [])) : null
         currentPlot.current = plot
         if (chart) {
+          marks.forEach(({ type }) => {
+            switch (type) {
+              case 'interval':
+                setChartTooltip(chart.interval(), tooltipCfg)
+                break
```

Comment:
Wondering if something like this would work:

```suggestion
            setChartTooltip(chart[type](), tooltipCfg)
```

Might need edgecase check for invalid type so that undefined() is not called.

Reply:
I was wondering if something like this exists in javascript. Now it can be written in one line like this: 
`if (type) chart[type]().position(tooltipCfg).tooltip(tooltipCfg)` 
And it works! Thanks.